app/files/in_memory_zip.py

```diff
@@ -0,0 +1,29 @@
+from io import BytesIO
+import zipfile
+
+
+class InMemoryZip(object):
+    def __init__(self):
+        # Create the in-memory file-like object
+        self.in_memory_zip = BytesIO()
+
+    def append(self, filename_in_zip, file_contents):
+        '''Appends a file with name filename_in_zip and contents of
+        file_contents to the in-memory zip.'''
+        # Get a handle to the in-memory zip in append mode
+        zf = zipfile.ZipFile(self.in_memory_zip, "a", zipfile.ZIP_STORED, False)
```

Comment:
should this be a context manager?

https://docs.python.org/2/library/zipfile.html#zipfile-objects